mne/viz/_brain/_brain.py

```diff
@@ -47,6 +47,157 @@ def safe_event(fun, *args, **kwargs):
         traceback.print_exc(file=sys.stderr)
 
 
+class _Overlay(object):
+    def __init__(self, scalars, colormap, rng, opacity):
+        self._scalars = scalars
+        self._colormap = colormap
+        self._rng = rng
+        self._opacity = opacity
+
+    def to_colors(self):
+        from matplotlib.cm import get_cmap
+        from matplotlib.colors import ListedColormap
+        if isinstance(self._colormap, str):
+            cmap = get_cmap(self._colormap)
+        else:
+            cmap = ListedColormap(self._colormap / 255.)
+
+        def diff(x):
+            return np.max(x) - np.min(x)
+
+        def norm(x, rng=None):
+            if rng is None:
+                rng = [np.min(x), np.max(x)]
+            return (x - rng[0]) / (rng[1] - rng[0])
+
+        rng = self._rng
+        scalars = self._scalars
+        if diff(scalars) != 0:
+            scalars = norm(scalars, rng)
+
+        colors = cmap(scalars)
+        if self._opacity is not None:
+            colors[:, 3] *= self._opacity
+        return colors
+
+
+class _LayeredMesh(object):
+    def __init__(self, renderer, vertices, triangles, normals):
+        self._renderer = renderer
+        self._vertices = vertices
+        self._triangles = triangles
+        self._normals = normals
+
+        self._polydata = None
+        self._actor = None
+        self._is_mapped = False
+
+        self._cache = None
+        self._overlays = dict()
```

Comment:
We still need to use `OrderedDict` until we switch to Python3.7 minimum (it was only an official language feature in 3.7, in 3.6 it works with standard Python but it was an implementation detail)